main/models.py

```diff
@@ -25,8 +25,8 @@ def __str__(self):
         return self.title
 
     def approved_comments(self):
-        """Approving comments. Use for moderating."""
-        return self.comments.filter(approved_comment=True)
+        """Will show comments that are ok in your opinion."""
+        return self.comments.get_approved()
```

Comment:
So I've googled a little, and recalled that this could be done using custom db manager. See the following for more info:
- https://docs.djangoproject.com/en/1.10/topics/db/managers/#modifying-a-manager-s-initial-queryset
- https://docs.djangoproject.com/en/1.10/topics/db/managers/#using-managers-for-related-object-access